### PR TITLE
Overhaul user manual DSpace page

### PR DESCRIPTION
**Pull request by sallain**
In this update I have:

* Updated screenshots
* Updated formatting
* Changed METS snippets from screenshots to literals
* Clarified transfer setup requirements
* Removed requirement to normalize for preservation (as far as I can tell, it works just fine if you do not normalize)
* And just done a general tidy-up of the page for tone, clarity, etc.

Connected to archivematica/Issues#384

### CODE REVIEW

File: user-manual/transfer/dspace.rst [hunk 1]
```diff
@@ -4,95 +4,119 @@
...
    :align: center
    :width: 80%
-   :alt:  A DSpace item export
+   :alt: An unzipped DSpace export package, showing the four files that comprise the item.
 
-* bitstream_39691_txt = the OCR text file
+In the example pictured above, the DSpace item includes four files:
 
-* bitstream_8272.pdf = the object deposited in DSpace
+* ``bitstream_39691_txt``: the OCR text file.
```
Comment (ablwr) [inline user-manual/transfer/dspace.rst]: I think you mean bitstream_39691.txt` here

**Review by ablwr — CHANGES_REQUESTED** [review]
Just this one tiny typo!

File: user-manual/transfer/dspace.rst (hunk 1 above)
Comment (sallain) [inline user-manual/transfer/dspace.rst]:
```suggestion
* ``bitstream_39691.txt``: the OCR text file.
```

**Comment by ablwr** [comment]
Sorry for not seeing where the "make a small change" github feature is or I could have made that recommendation myself!

**Review by ablwr — APPROVED** [review]

**Comment by sallain** [comment]
@ablwr no worries! When you leave a comment on a line, one of the formatting icons is for "Suggestion" - it's perfect for little errors like this (though it doesn't work for multi-line). Thanks for the review!

**Comment by ablwr** [comment]
Weird, is it only after you leave a comment? I don't see it when I am getting ready to add one. I'll give it a test next time.

**Comment by sallain** [comment]
If you're looking at the files on this PR, it might not show because the PR is merged (and therefore you can't commit anything else). It appears like this:

image

**Comment by ablwr** [comment]
Oooooooh THANK YOU!!! I have never looked up there.

**Pull request merged**